content/docs/command-reference/exp/show.md

```diff
@@ -74,7 +74,11 @@ will be generated using the same data from the table.
 - `-A`, `--all-commits` - include experiments derived from all Git commits, as
   well as from the last one. This prints all experiments in the project.
 
-- `-n <num>`, `--num <num>` - show the last `num` commits from HEAD.
+- `--rev <commit>` - show experiments derived from the specified `<commit>` as
+  baseline. Defaults to `HEAD` if none of `--rev`, `-a`, `-T`, `-A` is used.
+
+- `-n <num>`, `--num <num>` - show experiments from the `--rev` baseline and
+  from `num` commits before it. Give `-1` to include all the parent commits.
```

Comment:
Sounds like we're going for something similar to `git log -n` now per https://github.com/iterative/dvc/pull/7245#discussion_r797801555. So maybe

```suggestion
- `-n <num>`, `--num <num>` - show experiments from the `--rev` baseline and
  from `-num` commits before it. `num` accepts negative values (similar to
  `git log -n`).
```

Reply:
> We can wait until -1 has been decided.

Note that I removed that text since I assume it's no longer a function of `-n`.

Reply:
Two thoughts:

1. It's more accurate to say something like "show experiments from the previous `num` commits, starting from the `--rev` baseline." The `--rev` baseline is included in `-n`.
2. I don't know that referring to `git log` helps since I don't see that the negative value behavior is documented (and as you rightfully mention in https://github.com/iterative/dvc/pull/7245#discussion_r807429664, the defaults are different between `dvc exp show -n` and `git log -n`).

Reply:
And should `-num` be replaced by `--num` or maybe `<num>'? `-num`/`--num` I think it is only the flag name here?

Reply:
I agree with Dave's phrasing. And by `-num` I literally meant - (minus) `num` since the value is negative. I'll open another suggestion since this one was committed...

Reply:
Oh, what I mean here is what you suggestion "from `-num` commits before it." compare to my original "from `num` commits before it".